imkar/scattering/analytical.py

```diff
@@ -0,0 +1,186 @@
+import numpy as np
+import pyfar as pf
+
+import numbers
+
+
+def rectangular(frequencies, incident_angles, rectangle_width, gap_width,
+                height, speed_of_sound=346.18):
+    """
+    This function computes the scattering coefficient of periodic rectangular
+    profiles, assuming that the extent is infinite.
+
+    This function is based on Ducourneau's method described by Embrechts
+    [#]_.
+
+    Parameters
```

Comment:
How about adding a small image explaining the geometry? I assume it would be similar to Fig. 1 in Embrechts and Billon. Explicitly referenceing this image might help to easier understand the parameters.